xwiki-commons-core/xwiki-commons-velocity/src/main/java/org/xwiki/velocity/tools/EscapeTool.java

```diff
@@ -68,7 +69,7 @@
     @Override
     public String xml(Object content)
     {
-        return XMLUtils.escape(content);
+        return XMLUtils.escape(Objects.toString(content, null));
```

Comment:
Second argument is completely useless there no? According to javadoc it's exactly what does `Objects.toString` with only one argument.

Reply:
```
jshell> Objects.toString(null);
$1 ==> "null"

jshell> Objects.toString(null, null);
$2 ==> null
```

`Objects#toString(Object)` returns the `"null"` string in case of a null parameter.